application/src/test-fixtures/java/org/opentripplanner/gtfs/graphbuilder/GtfsBundleTestFactory.java

```diff
@@ -0,0 +1,23 @@
+package org.opentripplanner.gtfs.graphbuilder;
+
+import java.io.File;
+import javax.annotation.Nullable;
+import org.opentripplanner.datastore.api.DataSourceTestFactory;
+import org.opentripplanner.datastore.api.FileType;
+import org.opentripplanner.gtfs.config.GtfsDefaultParameters;
+
+public class GtfsBundleTestFactory {
+
+    public static GtfsBundle forTest(File gtfsFile, @Nullable String feedId) {
```

Comment:
Seems that prettier is not configured to format these files. Can you change that?